packages/hebao_v1/contracts/modules/security/GuardianUtils.sol

```diff
@@ -79,7 +61,7 @@ library GuardianUtils
         // Update the signingGuardians array with the actual number of guardians that have signed
         // (could be 1 less than the length if the owner signed as well)
         assembly { mstore(signingGuardians, numGuardians) }
-        GuardianGroupSizes memory signed = countGuardians(signingGuardians);
+        uint[16] memory signed = countGuardians(signingGuardians);
```

Comment:
maybe we should move the `SigRequirement.OwnerRequired` check to above, right after the for-loop.

Reply:
Will do to keep things closer apart, though it shouldn't matter because relayers/UI will check if these are valid before anything is submitted on-chain anyway, but a little bit of gas can indeed be saved in case things go wrong with this.